site/en/docs/extensions/mv3/linux_hosting/index.md

```diff
@@ -3,21 +3,23 @@ layout: "layouts/doc-post.njk"
 title: "Installing extensions on Linux"
 date: 2017-12-14
 updated: 2018-03-23
-description: How to package, host, and update crx files from a personal server.
+description: How to package, host, and update crx files from a personal server for Linux users.
 ---
 
 {% include 'partials/extensions/mv2page-in-mv3.md' %}
 
-Extensions hosted outside of the [Chrome Web Store][1] can only be installed by Linux users. This
-article describes how to package, host, and update `.crx` files from a personal server. If
-distributing an extension or theme solely through the [Chrome Web Store][2], consult [Webstore
-Hosting and Updating][3].
+Linux is the only platform where Chrome users can install extensions that are hosted outside of the
+[Chrome Web Store][chrome-web-store]. This article describes how to package, host, and update `.crx`
+files from a general purpose web server. If you are distributing an extension or theme solely
+through the [Chrome Web Store][chrome-web-store], consult [Webstore Hosting and
+Updating][hosting-options].
```

Comment:
```suggestion
through the [Chrome Web Store][chrome-web-store], consult [Webstore hosting and
updating][hosting-options].
```

Reply:
Since titles are sentence case, links that quote them should be sentence case as well. I'm afraid I haven't been good at checking these. Please keep an eye out for this.